pytorch/training/docker/1.7/py3/Dockerfile.cpu

```diff
@@ -122,6 +123,10 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
  && pip uninstall -y torchvision \
  && pip install --no-deps --no-cache-dir -U ${PT_TORCHVISION_URL}
 
+# Install PT S3 plugin
+RUN pip install -U ${PT_S3_WHL}
+RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
nit: Please break this into 2 different lines like other multi-command `RUN` statements.